exchanges/btcmarkets/btcmarkets.go

```diff
@@ -52,316 +60,482 @@ type BTCMarkets struct {
 
 // GetMarkets returns the BTCMarkets instruments
 func (b *BTCMarkets) GetMarkets() ([]Market, error) {
-	type marketsResp struct {
-		Response
-		Markets []Market `json:"markets"`
-	}
-
-	var resp marketsResp
-	path := fmt.Sprintf("%s/v2/market/active", b.API.Endpoints.URL)
-
-	err := b.SendHTTPRequest(path, &resp)
-	if err != nil {
-		return nil, err
-	}
-
-	if !resp.Success {
-		return nil, fmt.Errorf("%s unable to get markets: %s", b.Name, resp.ErrorMessage)
-	}
-
-	return resp.Markets, nil
+	var resp []Market
+	path := btcMarketsAPIURL + btcMarketsAPIVersion + btcMarketsAllMarkets
+	return resp, b.SendHTTPRequest(path, &resp)
```

Comment:
change path -> btcMarketsUnauthPath

Reply:
This can also be simplified in terms of just sending through the endpoint instead of constructing the path in and can move that to SendHTTPRequest